fedora/client/bodhi.py

```diff
@@ -142,7 +142,7 @@ def query(self, release=None, status=None, type_=None, bugs=None,
         auth = False
         if params['mine']:
             auth = True
-        for key, value in params.items():
+        for key, value in list(params.items()):
```

Comment:
The `list()` call here isn't really needed, since you just iterate over the items once. You'd need to do something like this in PY3 if you were assigning `params.items()` to a variable and would want to iterate over it multiple times.